googlemaps/googlemaps.php

```diff
@@ -23,24 +25,24 @@ function googlemaps_uninstall()
 	logger("removed googlemaps");
 }
 
-function googlemaps_location($a, &$item)
+function googlemaps_location(App $a, array &$item)
 {
 
-	if(! (strlen($item['location']) || strlen($item['coord']))) {
+	if (empty($item['location']) || empty($item['coord'])) {
```

Comment:
This should be `if (empty($item['location']) || !empty($item['coord'])) {` (notice the second condition).

Reply:
This is mandatory before merge as it is a regression error.